### ISSUE TITLE
CDC panic: unknown flag, please report a bug

### ISSUE THREAD

**Issue by Tammyxia** [pingcap/ticdc#1623]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

- Execute "$ tiup cdc:nightly cli changefeed update -c replication-task-1  --pd=http://172.31.10.94:2379 --config=changfeed.toml"
- $ cat changfeed.toml
[filter]
rules = ['rtest1', 'rtest2']

2. What did you expect to see?
As this doc: https://docs.pingcap.com/tidb/stable/manage-ticdc#create-a-replication-task
I expect the command has no any error, or report error with valid message.

3. What did you see instead?
![image](https://user-images.githubusercontent.com/78345569/114124275-a4f10d00-9926-11eb-891e-e7d1915a8ee2.png)

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
      v5.0.0-nightly, Git Commit Hash: 09a4c57d00175ae3ff9c900d5fb67f6757c58181
        ```

    - TiCDC version (execute `cdc version`):

        ```
       release-version=v5.0.0-master] [git-hash=b33b7fa15b4986db069cfc031bcd7fb827207b0b]
        ```

**Comment by amyangfei** [comment]
Have been fixed in https://github.com/pingcap/ticdc/pull/1596, could you upgrade to latest version and confirm it? @Tammyxia

**Comment by Tammyxia** [comment]
fixed in cdc 5.0.0 release
